people.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "nick": "Undefine",
+    "coordinates": [50.01052, 22.682],
+    "links": {
+      "GitHub": "https://github.com/Undefine-Unity"
+    },
+    "contributor": true
```

Comment:
sorry, I don't see any commits from you in the serenity repo... https://github.com/SerenityOS/serenity/commits?author=Undefine-Unity